commit 73af779d314902f2af9d62e86a5adb17fb9f5d00
Author: Sage Weil <sweil@redhat.com>
Date:   Fri Apr 16 14:58:34 2021 -0400

    mgr/MgrStandby: fix config observer
    
    Otherwise handle_conf_change() is never called.
    
    Signed-off-by: Sage Weil <sage@newdream.net>

diff --git a/src/mgr/MgrStandby.cc b/src/mgr/MgrStandby.cc
index b78874c05ff..1ba6eaea4f6 100644
--- a/src/mgr/MgrStandby.cc
+++ b/src/mgr/MgrStandby.cc
@@ -116,6 +116,8 @@ int MgrStandby::init()
   init_async_signal_handler();
   register_async_signal_handler(SIGHUP, sighup_handler);
 
+  cct->_conf.add_observer(this);
+
   std::lock_guard l(lock);
 
   // Start finisher
